lib/mapObjects/CGTownInstance.cpp

```diff
@@ -1134,6 +1134,7 @@ void CGTownInstance::serializeJsonOptions(JsonSerializeFormat & handler)
 		eventsHandler.syncSize(events, JsonNode::JsonType::DATA_VECTOR);
 		eventsHandler.serializeStruct(events);
 	}
+	handler.serializeId("alignmentToPlayer", alignmentToPlayer, PlayerColor::NEUTRAL);
```

Comment:
@IvanSavenko please verify that this is ok

Reply:
Looks OK. And since this method is only used for editor / vmap's it can't break base game.